src/com/palmergames/bukkit/config/ConfigNodes.java

```diff
@@ -1678,6 +1683,15 @@
 			"250.0",
 			"",
 			"# How much it costs to start a town."),
+	ECO_PRICE_TOWN_MERGE(
+			"economy.new_expand.price_town_merge",
+			"0",
+			"# The base cost a town has to pay to merge with another town."),
+	ECO_PRICE_TOWN_MERGE_PER_PLOT_PERCENTAGE(
+			"economy.new_expand.price_town_merge_per_plot_percentage",
+			"50",
+			"# The percentage that a town has to pay per plot to merge with another town.",

```

Comment:
Might be nice to be clear at this point which town will pay the cost. Same for the price_town_merge.